### PR TITLE
toipe: add page

### PR DESCRIPTION
**Pull request by TechnoFlask**
Toipe is a terminal typing test built in rust.

- [x] The page(s) are in the correct platform directories: `common`
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/toipe.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+> A trusty terminal typing tester.
+> More information: <https://github.com/Samyak2/toipe>.
+
+- Start typing test with default wordlist:
+
+`toipe`
+
+- Use a specific wordlist:
+
+`toipe -w/--wordlist {{wordlist_name}}`
```
Comment (spageektti) [inline pages/common/toipe.md:13]:
```suggestion
`toipe {{-w|--wordlist}} {{wordlist_name}}`
```

File: pages/common/toipe.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+- Use a custom wordlist:
+
+`toipe -f/--file {{file_path}}`
```
Comment (spageektti) [inline pages/common/toipe.md:17]: [Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax)
```suggestion
`toipe {{-f|--file}} {{file_path}}`
```

File: pages/common/toipe.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Specify number of words on each test:
+
+`toipe -n/--num {{number_of_words}}`
```
Comment (spageektti) [inline pages/common/toipe.md:21]:
```suggestion
`toipe {{-n|--num}} {{number_of_words}}`
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution and welcome to TLDR pages. Can you please sign the CLA?

File: pages/common/toipe.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# toipe
+
+> Yet another typing test, but crab flavoured.
+> A trusty terminal typing tester.
+> More information: <https://github.com/Samyak2/toipe>.
+
+- Start typing test with default wordlist:
```
Comment (kbdharun) [inline pages/common/toipe.md:7]:
```suggestion
- Start typing test with the default wordlist:
```

File: pages/common/toipe.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`toipe`
+
+- Use a specific wordlist:
+
+`toipe {{-w|--wordlist}} {{wordlist_name}}`
+
+- Use a custom wordlist:
+
+`toipe {{-f|--file}} {{file_path}}`
```
Comment (kbdharun) [inline pages/common/toipe.md:17]:
```suggestion
`toipe {{-f|--file}} {{path/to/file}}`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have some minor suggestions for this page.

File: pages/common/toipe.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+- Use a specific wordlist:
+
+`toipe {{-w|--wordlist}} {{wordlist_name}}`
+
+- Use a custom wordlist:
+
+`toipe {{-f|--file}} {{path/to/file}}`
+
+- Specify number of words on each test:
```
Comment (kbdharun) [inline pages/common/toipe.md:19]:
```suggestion
- Specify the number of words on each test:
```

**Review by kbdharun — APPROVED** [review]

**Review by spageektti — APPROVED** [review]
LGTM, thanks for your contribution!

**Pull request merged**